### PR TITLE
LRAC-12833 | master

### PR DESCRIPTION
**Pull request by JLucasM**
Jira ticket: https://issues.liferay.com/browse/LRAC-12833

### CODE REVIEW

**Comment by JLucasM** [comment]
ci:test:sf

**Comment by JLucasM** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
wait for https://github.com/brianchandotcom/liferay-portal/pull/129466 being merged

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx [hunk 1]
```diff
@@ -27,6 +28,10 @@ const response = {
 	product: 0,
 };
 
+// const request = {
```
Comment (marcosapmf) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx:31]: Remove

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx [hunk 2]
```diff
@@ -66,6 +73,8 @@ describe('Attributes Step', () => {
 			</AppContextProvider>
 		);
 
+		debug();
```
Comment (marcosapmf) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx:76]: Remove

**Review by marcosapmf — CHANGES_REQUESTED** [review]

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx (hunk 1 above)
Comment (JLucasM) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx]: Done!

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx (hunk 2 above)
Comment (JLucasM) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/__tests__/AttributesStep.spec.tsx]: Done! thanks @marcosapmf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**